### DISCUSSION TITLE
A global keyboard shortcut to enable / disable the program

### DISCUSSION THREAD

**Discussion by sepsol** [mulaRahul/keyviz#66] [Ideas]
It would be nice if there was an option to set a keyboard shortcut for when we want to enable/disable this program. Because in some software we might have keyboard shortcuts that use `Shift` or even in the example of VS Code, we might have keyboard shortcuts that are normal keys like `Ctrl + K` `F`. 

In these cases, I think it'd be great if the user could enable/disable the program on-demand with a global shortcut key (which should probably be ignored and not displayed when the user tries to disable the program).

**Comment by sepsol** [comment]
Now that I think about it, it might be even better to have a global keyboard shortcut that enables/disables the "Hotkey Filter" settings since enabling/disabling the whole program would become unmanageable quickly. Although having both options would be even better.

**Reply by mulaRahul** [reply]
Yes, @sepsol, it's a heavily requested feature (#35), so you'll see both the hot-key filter & global visualizer on/off keyboard shortcuts and others in the upcoming release.

**Reply by sepsol** [reply]
That's great to hear, thanks!

**Comment by mulaRahul** [comment]
Keystroke visualizer toggle hotkey added with [v2.0.0a](https://github.com/mulaRahul/keyviz/releases/tag/v2.0.0a).

**Reply by sepsol** [reply]
Awesome job, I really like where this is going! 

Do you think it would also be possible to have a toggle just for the "Hotkey Filter" as well as the "global keystroke visualizer" on/off keyboard shortcuts? 

While testing the new mouse events I noticed that it didn't capture the double-clicks quite well. It would also be nice if the right-click and left click where distinguishable and that the mouse-wheel event showed the direction and was positioned besides the cursor.
